app/models/comment.rb

```diff
@@ -1,2 +1,8 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_post_comments_counter
+    post.update(comments_counter: post.comments.count)
```

Comment:
[OPTIONAL] I think your code will be more concise if you had just done this  post.increment!(:comments_counter). You can also include a after_save callback and make the method private this is a good practice.

Reply:
Done

Reply:
Done